docs/docs/userguide/configuration/BITBUCKET_CONFIGURATION.md

```diff
@@ -18,6 +18,21 @@ For more information, see [Bitbucket Version Tags](https://hub.docker.com/r/atla
 bitbucket_version_tag = "<BITBUCKET_VERSION_TAG>"
 ```
 
+### Number of Bitbucket application nodes
+
```

Comment:
If you desire to install more than one application node, you must include **System Admin Credentials** and **License** properties listed below in the first installation.